tests/tests_basic.yml

```diff
@@ -0,0 +1,99 @@
+- name: Basic snapshot test
+  hosts: all
+  vars:
+    - listA: []
+    - listB: []
+    - listC: []
+
+  tasks:
+
+    - name: Run the storage role to create test LVs
+      include_role:
+        name: fedora.linux_system_roles.storage
+
+    - name: Get unused disks
+      include_tasks: get_unused_disk.yml
+      vars:
+        min_size: "1g"
+        min_return: 10
+
+    - block:
+        - debug:
+            var: listA
+        - debug:
+            var: listB
+        - debug:
+            var: listC
+        - set_fact:
+            listA: "{{ listA + [ unused_disks[item|int] ] }}"
+          with_sequence: start=0 end=2 stride=1
+        - set_fact:
+            listB: "{{ listB + [ unused_disks[item|int] ] }}"
+          with_sequence: start=3 end=5 stride=1
+        - set_fact:
+            listC: "{{ listC + [ unused_disks[item|int] ] }}"
+          with_sequence: start=6 end=9 stride=1
```

Comment:
If you need to test Ansible/Jinja code, you can create small playbooks like this:
```yaml
- name: Test something
  hosts: localhost
  gather_facts: false  # much faster if you do not need facts such as distribution, os_family, etc.
  vars:
    listA: []
    ...
  tasks:
    - name: See what list addition looks like
      debug:
        msg: add {{ (listA + [something]) | to_nice_json }}
...
```

then run it like this:

`ansible-playbook -vv my-test.yml`

This is really helpful when trying to figure out how to do something with a loop/filter.  See also  https://linux-system-roles.github.io/documentation/howto/working-with-ansible-jinja-code-and-filters.html and https://docs.ansible.com/ansible/latest/playbook_guide/complex_data_manipulation.html

Reply:
Thanks for the tips.

Are you requesting I change how the lists are split?

Reply:
> Thanks for the tips.
> 
> Are you requesting I change how the lists are split?

Not necessarily, just wanted to offer some suggestions